### ISSUE TITLE
Suggestion - rename "Johnson and Johnson" file to something informative + A "run me"

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [nngogol/PySimpleGUIDocGen#45]
## A big Johnson?

I saw this file in my folder and didn't understand its purpose.  Is it a temp file?  I ended up deleting it after I did a search of the code.

## `run_me.py`
[run_me.py.txt](https://github.com/nngogol/PySimpleGUIDocGen/files/3493475/run_me.py.txt)

I ended up creating a filenamed this way because I was not able to remember exactly what to call and with what options.  It's been a LOT easier to run this file and to look at this output it creatss:

![image](https://user-images.githubusercontent.com/13696193/62881794-904a1500-bcfe-11e9-9154-cace298d8282.png)

As you can see, it SAVED MY ASS in this last run because I have a missing method. I deleted it from the code, but did not fix the .md file, yet.  Thanks to your dumped output I'm able to feel very **confident** in the quality and completeness of my readme. I do not hesitate to copy it to my main folder and release it to GitHub and ReadTheDocs.

In short, I can't live without this tool!  The entire package has benefited GREATLY from it!  Thank you so much for making it!!!
